src/home_robot/home_robot/navigation_planner/discrete_planner.py

```diff
@@ -287,7 +286,7 @@ def plan(
                 #     print("Nowhere left to explore. Stopping.")
```

Comment:
should this still be commented out?

Reply:
I think this need not be commented out. But I will remove this after testing in a future PR along with other changes (for error reporting).